### DISCUSSION TITLE
Support Apple Arm

### DISCUSSION THREAD

**Discussion by chenrui333** [github/gh-actions-importer#61] [Feature requests] [answered]
Looks like the container image is not native to apple arm

```
$ gh actions-importer migrate travis-ci --target-url xxx --output-dir . --travis-ci-repository xxx
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

**Comment by Chaseshak** [comment, marked as answer]
Thanks for the request!

The [latest container image](https://github.com/actions-importer/preview/pkgs/container/cli/75925607?tag=latest) now supports both `linux/arm64` and `linux/amd64`.

Please run `gh actions-importer update` which will pull the latest image for your architecture.
